src/utils/response.ts

```diff
@@ -0,0 +1,12 @@
+export interface LambdaResponse {
+  batchItemFailures?: { itemIdentifier: string }[];
+}
+
+export function isBatchItemFailure(lambdaResponse: any): lambdaResponse is LambdaResponse {
```

Comment:
`lambdaResponse is LambdaResponse` is not needed here, not sure what it's trying to do but this function returns a boolean.